solr/contrib/scripting/src/test/org/apache/solr/scripting/handler/XSLTOutputWriterTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.scripting.handler;
+
+import org.apache.solr.SolrTestCaseJ4;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/** Tests the ability to configure multiple query output writers, and select those
+ * at query time.  This is specific to the XSLT writer, which isn't part of the core.
+ * 
+ * See the related unit test OutputWriterTest.
+ *
+ */
+public class XSLTOutputWriterTest extends SolrTestCaseJ4 {
```

Comment:
The formatting in this file is a bit off (indentation varies etc.).  Lucene recently adopted the Google Java Format.  I use an IntelliJ plugin for this.  Can you please reformat this file with that?  Even the built-in formatter is probably fine.

Reply:
I found this blog post: http://www.practicesofmastery.com/post/eclipse-google-java-style-guide/ (since I use Eclipse)...  Do we have a JIRA about moving towards this?   Should I look to figure out how to add the Google Java Format to the default Eclipse setup?  Or at least a message when you run `./gradlew eclipse` that recommends adding this formatter to your setup?

Reply:
Apparently there's an Eclipse plugin listed here: https://github.com/google/google-java-format
The Lucene side issue is here: https://issues.apache.org/jira/browse/LUCENE-9564
But you'll see in the comments were held off on Solr for a later date.

BTW if I'm mistaken and you moved an existing source file, then I recommend doing no formatting changes as it may fool git rename detection.

Reply:
Your recent edits here look good BTW.